interfaces/builtin/desktop_contacts_service.go

```diff
@@ -0,0 +1,157 @@
+dbus (receive, send)
+	bus=session
+	interface=org.freedesktop.DBus.Properties
+	path=/org/gnome/evolution/dataserver/AddressBookFactory
+	peer=(label=unconfined),
+dbus (receive, send)
+	bus=session
+	interface=org.freedesktop.DBus.Properties
+	path=/org/gnome/evolution/dataserver/AddressBookCursor{,/**}
+	peer=(label=unconfined),
+dbus (receive, send)
+	bus=session
+	interface=org.freedesktop.DBus.Properties
+	path=/org/gnome/evolution/dataserver/AddressBookView{,/**}
+	peer=(label=unconfined),
+dbus (receive, send)
+	bus=session
+	interface=org.freedesktop.DBus.Properties
+	path=/org/gnome/evolution/dataserver/Subprocess{,/**}
+	peer=(label=unconfined),
+# Allow access to methods
+dbus (receive, send)
+	bus=session
+	interface=org.gnome.evolution.dataserver.AddressBook
+	path=/org/gnome/evolution/dataserver/{Subprocess,AddressBook}{,/**}
+	peer=(label=unconfined),
+dbus (receive, send)
+	bus=session
+	interface=org.gnome.evolution.dataserver.AddressBookFactory
+	path=/org/gnome/evolution/dataserver/AddressBookFactory
+	peer=(label=unconfined),
+dbus (receive, send)
+	bus=session
+	interface=org.gnome.evolution.dataserver.AddressBookCursor
+	path=/org/gnome/evolution/dataserver/AddressBookCursor{,/**}
+	peer=(label=unconfined),
+dbus (receive, send)
+	bus=session
+	interface=org.gnome.evolution.dataserver.AddressBookView
+	path=/org/gnome/evolution/dataserver/AddressBookView{,/**}
+	peer=(label=unconfined),
+dbus (receive, send)
+	bus=session
+	interface=org.gnome.evolution.dataserver.Source
+	path=/org/gnome/evolution/dataserver/SourceManager{,/**}
+	peer=(label=unconfined),
+dbus (receive, send)
+	bus=session
+	interface=org.gnome.evolution.dataserver.Source.Removable
+	path=/org/gnome/evolution/dataserver/SourceManager{,/**}
+	peer=(label=unconfined),
+dbus (receive, send)
+	bus=session
+	interface=org.gnome.evolution.dataserver.SourceManager
+	path=/org/gnome/evolution/dataserver/SourceManager
+	peer=(label=unconfined),
+
+# Allow clients to introspect the service
+dbus (send)
+	bus=session
+	interface=org.freedesktop.DBus.Introspectable
+	path=/org/gnome/evolution/dataserver/AddressBook{,/**}
+	member=Introspect
+	peer=(label=unconfined),
+dbus (send)
+	bus=session
+	interface=org.freedesktop.DBus.Introspectable
+	path=/org/gnome/evolution/dataserver/AddressBookFactory
+	member=Introspect
+	peer=(label=unconfined),
+dbus (send)
+	bus=session
+	interface=org.freedesktop.DBus.Introspectable
+	path=/org/gnome/evolution/dataserver/AddressBookCursor{,/**}
+	member=Introspect
+	peer=(label=unconfined),
+dbus (send)
+	bus=session
+	interface=org.freedesktop.DBus.Introspectable
+	path=/org/gnome/evolution/dataserver/AddressBookView{,/**}
+	member=Introspect
+	peer=(label=unconfined),
+dbus (send)
+	bus=session
+	interface=org.freedesktop.DBus.Introspectable
+	path=/org/gnome/evolution/dataserver/SourceManager{,/**}
+	member=Introspect
+	peer=(label=unconfined),
+
+# Allow access to cached avatars
+owner @{HOME}/.cache/evolution/addressbook/** r,
```

Comment:
There is a 'trash' directory in @{HOME}/.cache/evolution/addressbook/. Perhaps this instead:
```
owner @{HOME}/.cache/evolution/addressbook/[0-9a-f]*/*.jpeg r,
```

There is a 'cache.db' file in there, so feel free to add this if needed:
```
owner @{HOME}/.cache/evolution/addressbook/[0-9a-f]*/cache.db r,
```

Alternatively, instead of the above two rules we could allow 'not trash', but I prefer the explicitness of the above.